docker/docker-compose.yml

```diff
@@ -27,14 +27,14 @@ services:
     deploy: *default-deploy
     volumes:
       - influxdb:/var/lib/influxdb/
-    healthcheck:
-      test: ["CMD-SHELL", "influx -execute 'quit'"]
-      interval: 5s
-      timeout: 5s
-      retries: 5
-      start_period: 10s
+      - "./influxdb/setup-influxdb.sh:/docker-entrypoint-initdb.d/setup-influxdb.sh:ro"
```

Comment:
Is the only reason you are creating a volume mount here to avoid a new dockerfile?

Reply:
Both postgres and influx database images have a `docker-entrypoint-initdb.d` folder and any script that is added to this folder will be executed during initialization. This offers a good way to run any setup code. In our case, we want to do a couple of things:
1. Wait for the service to be ready
2. Create two databases

To answer your question, that mount does remove the need for a dockerfile. That being said, one of the approaches I took was to create a docker file, but I quickly realized that was just one more image that needed to be managed. My goal is to simplify the docker process as much as possible and I found that I can do that by:
1. Re-using config files between RPM and docker setup
2. Integrate setup scripts into entrypoints when available 
3. Utilize existing images when possible

Reply:
I'm not sure another runtime volume is better than a new dockerfile that bakes the script into the image itself.